client/src/main/scala/drt/client/components/DropInComponent.scala

```diff
@@ -31,23 +31,20 @@ object DropInComponent extends WithScalaCssImplicits {
       dropInRegistrations = model.dropInRegistrations
     ))
 
-    private def roundToNearestHalfHour(hours: Double): Double = {
-      val wholeHours = hours.toInt
-      val fractionalHours = hours - wholeHours
+    def formatDuration(minutes: Int): String = {
```

Comment:
Think it would be good to have some tests around this function to make sure it copes with a bunch of cases